packages/web-pkg/src/components/SearchBarFilter.vue

```diff
@@ -122,7 +122,7 @@ export default defineComponent({
       emit('update:modelValue', { value: option })
     }
 
-    return { currentSelection, isIndexGreaterZero, onOptionSelected, locationOptions }
+    return { currentSelection, isIndexGreaterZero, onOptionSelected, locationOptions, $gettext }
```

Comment:
No need to return it explicitly.

```suggestion
    return { currentSelection, isIndexGreaterZero, onOptionSelected, locationOptions}
```